### PR TITLE
Add queries tag to support api lvl 30

### PR DESCRIPTION
**Pull request by erikeelde**
https://medium.com/androiddevelopers/package-visibility-in-android-11-cc857f221cd9
https://developer.android.google.cn/training/basics/intents/package-visibility

### CODE REVIEW

File: toggles-sample/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -3,6 +3,13 @@
     xmlns:tools="http://schemas.android.com/tools"
     package="com.example.wrench">
 
+<!--    <queries>-->
```
Comment (tingnapianhai) [inline toggles-sample/src/main/AndroidManifest.xml:6]: just curious why this change if the code not gonna to be in use ? 😄

Comment (erikeelde) [reply toggles-sample/src/main/AndroidManifest.xml:6]: I added it when trying to figure out why the heck everything stopped working.. :) was blissfully unaware of the changes made.. I'll make sure to remove it once I get around to take a next step with this PR (updated values notifications aren't propagating properly either - seems that's been changed too)

Comment (tingnapianhai) [reply toggles-sample/src/main/AndroidManifest.xml:6]: I see, nice 👍

Comment (erikeelde) [reply toggles-sample/src/main/AndroidManifest.xml]: gone!

**Review by tingnapianhai — APPROVED** [review]
👍 🚀

**Comment by erikeelde** [comment]
@tingnapianhai 👀 ❓ 😄

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,5 @@
-#Mon Feb 11 23:37:42 CET 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.7-rc-4-all.zip
```
Comment (tingnapianhai) [inline gradle/wrapper/gradle-wrapper.properties:3]: there is the release version now for 6.7 -> `gradle-6.7-all.zip` 😉

**Review by tingnapianhai — APPROVED** [review]

File: gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (erikeelde) [reply gradle/wrapper/gradle-wrapper.properties:3]: I know! I hoped to get away with it! :)
